src/WebJobs.Extensions/Extensions/Warmup/Trigger/WarmupTriggerAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Description;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Warmup.Trigger
```

Comment:
For intellisense discoverability, all binding attributes should be in the root Microsoft.Azure.WebJobs namespace. See other attributes.